grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientCompressionTest.java

```diff
@@ -50,6 +51,7 @@ protected void configure(ServerBuilder sb) {
             sb.service(GrpcService.builder()
                                   .addService(new TestServiceImpl(Executors.newSingleThreadScheduledExecutor()))
```

Comment:
Should we now test streaming requests and responses here as well?

Reply:
The logic is still handled in one place, `AbstractServerCall.doSendHeaders()`, but it would be good to add tests just in case.

Reply:
Added at [`dcae4a7` (#4192)](https://github.com/line/armeria/pull/4192/commits/dcae4a747ff83c209af08e20201a346976e17238)